### PR TITLE
METRON-1785 Automate deployment of packet capture for development environment

### PR DESCRIPTION
**Pull request by nickwallen**
While trying to test #1201 , I fixed some issues with the Ansible install of the components required for testing packet capture.  I added instructions for how to do this in the README.

## Testing

1. Spin-up the development environment and validate that alerts are visible in the Alerts UI and run the Metron Service Check in Ambari.

1. Follow the instruction in the README, to install and start all of the components for capturing packets.  Ensure that you can search and find these packets using the Alerts UI > PCAP tab.

## Pull Request Checklist

- [x] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel).
- [x] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?
- [x] Have you included steps to reproduce the behavior or problem that is being changed or addressed?
- [x] Have you included steps or a guide to how the change may be verified and tested manually?
- [x] Have you ensured that the full suite of tests and checks have been executed in the root metron folder via:
- [x] Have you written or updated unit tests and or integration tests to verify your changes?
- [x] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [x] Have you verified the basic functionality of the build by building and running locally with Vagrant full-dev environment or the equivalent?

### CODE REVIEW

File: metron-deployment/ansible/playbooks/sensor_install.yml [hunk 1]
```diff
@@ -46,6 +46,19 @@
...
+#
+- hosts: sensors
+  become: true
+  roles:
+    - { role: ambari_gather_facts }
+    - { role: tap_interface }
+    - { role: pcap_replay }
+    - { role: pycapa }
+  tags:
+    - pcap
```
Comment (nickwallen) [inline metron-deployment/ansible/playbooks/sensor_install.yml]: Allows us to deploy all of the components required for testing packet capture using the tag 'pcap'.

File: metron-deployment/ansible/roles/librdkafka/defaults/main.yml [hunk 2]
```diff
@@ -15,6 +15,6 @@
 #  limitations under the License.
 #
 ---
-librdkafka_version: 0.9.4
-librdkafka_url: https://github.com/edenhill/librdkafka/archive/v0.9.4.tar.gz
+librdkafka_version: 0.11.5
+librdkafka_url: https://github.com/edenhill/librdkafka/archive/v0.11.5.tar.gz
```
Comment (nickwallen) [inline metron-deployment/ansible/roles/librdkafka/defaults/main.yml]: Pycapa only works with 0.11.5 now.  This was tested and validated in a previous PR #1136.

File: metron-deployment/development/centos6/README.md [hunk 3]
```diff
@@ -78,28 +80,103 @@ Any platform that supports these tools is suitable, but the following instructio
...
+If you have already built Metron, you can skip the build step when deploying the development environment.
+
+  1. Deploy the development environment without re-building Metron.
+      ```
+      vagrant --ansible-skip-tags="build,sensors,pcap" up
+      ```
+
+### Deploy Packet Capture
+
+If you want to deploy the components required to generate and capture network packets.
```
Comment (nickwallen) [inline metron-deployment/development/centos6/README.md]: Specific instructions for deploying the components required to test packet capture in the development environment.

**Comment by MohanDV** [comment]
Deployed a full dev , I executed set of instructions as in Docs 

`vagrant up
vagrant --ansible-tags="pcap" provision`

 #Stopped the Parser, Enrichment, Indexing, and Profiler topologies to free-up resources.

`vagrant ssh
sudo su -
source /etc/default/metron
yum -y install wireshark`

I see that the pcap-replay and pycapa services not deployed ..

`[root@node1 ~]# service pcap-replay start`

`pcap-replay: unrecognized service` 

`[root@node1 ~]# service pycapa start`

`pycapa: unrecognized service`

File: metron-deployment/ansible/roles/librdkafka/defaults/main.yml (hunk 2 above)
Comment (JonZeolla) [reply metron-deployment/ansible/roles/librdkafka/defaults/main.yml]: Should we update the bro package as well?  I would need to double check but I think it uses the version of librdkafka installed here in full-dev, but the package docs say "This plugin has been tested against the latest release of librdkafka, which at the time of this writing is v0.9.4."

**Comment by nickwallen** [comment]
Yes, you are right @MohanDV.  Somehow the default tags are interacting badly with the tags that are passed in.  I'll try to figure out what's going on.

**Pull request closed without merging**